src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -323,6 +323,8 @@ private class Emit(
                 .invoke[String, String]("concat", xi.load().toS)
                 .invoke[String, String]("concat", " / ")
                 .invoke[String, String]("concat", len.load().toS)
+                .invoke[String, String]("concat", ". IR: ")
+                .invoke[String, String]("concat", Pretty(x))
```

Comment:
this is going to be sad if the size of `Pretty(x)` gets to be larger than 65535. Hopefully never, but just fyi.

Reply:
also, I wrote this before I wrote `CodeString.concat()` and never got around to fixing it, but you can just do `.concat(Pretty(x))` now.